qa/L0_batcher/batcher_test.py

```diff
@@ -182,14 +182,15 @@ def check_response(self,
         except Exception as ex:
             self.add_deferred_exception(ex)
 
-    def check_setup(self, model_name):
+    def check_setup(self, model_name, preferred_batch_sizes,
+                    max_queue_delay_us):
         # Make sure test.sh set up the correct batcher settings
         config = self.triton_client_.get_model_config(model_name).config
         bconfig = config.dynamic_batching
-        self.assertTrue(2 in bconfig.preferred_batch_size)
-        self.assertTrue(6 in bconfig.preferred_batch_size)
+        for i in preferred_batch_sizes:
```

Comment:
You should also check if len of preferred_batch_sizes matches the len of same in bconfig